src/util/link-helper.ts

```diff
@@ -39,6 +39,9 @@ function hasKnownProtocol(input: string) {
         'https',
         'http',
 
+        // `file` may or may not work, due to cross-origin restrictions and browser settings.
+        'file',
```

Comment:
Using `file://` links probably won't actually work for people, but by adding them here, lime-elements won't add `https://` in front of them, which at least make people see that their browser is blocking the request, rather than ping us about limel-input-field's weird behaviour 😅

See [thread on our forum](https://forum.lime-crm.com/t/urlastext-still-adding-https-before-my-url/5648/4?u=ads) (internal link)